services/email/index.js

```diff
@@ -5,7 +5,7 @@ var nodemailer = require('nodemailer');
 var config = require('../../config');
 var i18nFuture = require('hof').i18n;
 var Hogan = require('hogan.js');
-var i18nLookup = require('hof').i18nLookup;
+var i18nLookup = require('i18n-lookup');
```

Comment:
i18n is a node implementation of i18n - the core translation library, i18n-lookup just accepts an array of keys and looks up the first one that matches. Just a utility function that was probably abstracted a bit too far IMO